course/assets/js/multiple_choice.js

```diff
@@ -0,0 +1,35 @@
+export const clearSolution = () => {
+  const radioElements = document.getElementsByClassName("multiple-choice-radio")
+  for (const radio of radioElements) {
+    radio.addEventListener("click", () => {
+      const solutions = radio
+        .closest("fieldset")
+        .getElementsByClassName("correctness-icon")
+
+      for (const solution of solutions) {
+        solution.classList.remove("toggle-visible")
+      }
+    })
+  }
+}
+
+export const checkAnswer = () => {
+  const buttons = document.getElementsByClassName(
```

Comment:
Similarly to above, I think we can avoid declaring the extra constant and loop here and instead just use:

```
  Array.from(
    document.getElementsByClassName("multiple-choice-check-button")
  ).forEach(button => {
    button.addEventListener("click", () => {
      Array.from(
        button
          .closest(".multiple-choice-question")
          .getElementsByClassName("multiple-choice-radio")
      ).forEach(input => {
        if (input.checked) {
          const solution = input
            .closest("div")
            .getElementsByClassName("correctness-icon")[0]
          solution.classList.add("toggle-visible")
        }
      })
    })
  })
```